### PR TITLE
chore: configure release-please for monorepo

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Docs: https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md

Since now we have several apps and libs, it makes sense to use one release flow for each project.

I've added test commit "fix(permit-utils): use const for default permit gas limit" just to check if `release-please` works well.

Test results: https://github.com/cowprotocol/cowswap/pull/3601/files

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "bootstrap-sha": "385417f5afd830a62a0b06c9e620a78be3c1ccdd",
```
Comment (shoom3301) [inline release-please-config.json:2]: Last commit in `main` branch

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "cowswap",
-  "version": "1.54.2",
-  "description": "CoW Swap",
+  "name": "cowswap-monorepo",
+  "version": "2.0.0",
+  "description": "CoW Swap Protocol",
```
Comment (alfetopito) [inline package.json:4]: Hmm, either keep cow swap or switch to cow protocol.

File: release-please-config.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "apps/cowswap-frontend": {
+      "changelog-path": "CHANGELOG.md",
+      "release-type": "node",
+      "bump-minor-pre-major": false,
+      "bump-patch-for-minor-pre-major": false,
+      "draft": false,
+      "prerelease": false
+    },
+    "apps/explorer": {
+      "changelog-path": "CHANGELOG.md",
```
Comment (alfetopito) [inline release-please-config.json:13]: Does release please create the file if it's missing?

Comment (alfetopito) [reply release-please-config.json:13]: I saw it does in #3601

File: package.json (hunk 2 above)
Comment (shoom3301) [reply package.json:4]: Changed to "CoW Protocol", thanks!

**Review by alfetopito — APPROVED** [review]

**Pull request merged**